entitlement/src/main/java/org/killbill/billing/entitlement/api/DefaultEntitlementContext.java

```diff
@@ -122,7 +120,9 @@ public DefaultBaseEntitlementWithAddOnsSpecifier apply(final BaseEntitlementWith
     private static <T> Iterable<T> lastUnlessEmptyOrFirst(final Iterable<T> prevValues, final Iterable<T> newValues) {
         // Be lenient if a plugin returns an empty list (default behavior for Ruby plugins): at this point,
         // we know the isAborted flag hasn't been set, so let's assume the user actually wants to use the previous list
-        if (newValues != null && !Iterables.isEmpty(newValues)) {
+        // FIXME-1615 : Instead of "iterableToList(newValues).isEmpty()", should we copy guava version to new class
+        //  "CollectionUtils.isIterableEmpty(Iterable)" ?
```

Comment:
👍

Reply:
Ok, instead of `CollectionUtils`, maybe will changed to our future `Iterables.isEmpty()` (as discussed by me and @sbrossie in slack)